variables.env

```diff
@@ -16,24 +16,33 @@ WMDE_RELEASE_VERSION=main
 # Update only patch versions for security releases.
 # Choose latest version for major releases.
 # https://releases.wikimedia.org/mediawiki/
+WBS_WIKIBASE_VERSION=3.0.0
 MEDIAWIKI_VERSION=1.42.0-rc.0
 
 
 # ##############################################################################
 # Wikidata Query Service Version
 # ##############################################################################
+# Update to reflect upstream changes
+WBS_WDQS_VERSION=1.0.0
+
 # Does not follow laws of god or men.
 # Update to the latest version for major releases.
 # Reason about changes for patch versions.
 # as per https://github.com/wikimedia/wikidata-query-rdf/tags
 # and https://archiva.wikimedia.org/repository/releases/org/wikidata/query/rdf/service/
 # Compare to previous version: https://github.com/wikimedia/wikidata-query-rdf/compare/query-service-parent-0.3.135...query-service-parent-0.3.137
 WDQS_VERSION=0.3.137
+WBS_WDQS_FRONTED_VERSION=1.0.0
+WBS_WDQS_PROXY_VERSION=1.0.0
```

Comment:
# ##############################################################################
# Wikidata Query Service Proxy Version
# ##############################################################################
# Update to reflect changes